### PR TITLE
W-14222920-ping-cloud-duke

### PR DESCRIPTION
**Pull request by dukesphere**

### CODE REVIEW

File: modules/ROOT/pages/_partials/acb-reusable-steps.adoc [hunk 1]
```diff
@@ -12,7 +12,18 @@
...
 ====
 // end::open-command-palette[]
+//
+//
+// tag::check-connection-http[]
+. Before starting your Mule app, confirm that the connection to the listener is successful: 
++
+From the configuration XML, above the HTTP Listener configuration (`<<http:listener-config/>`), click *Test Connection*.
 
+* If the test succeeds, *Connection is valid* appears in a popup message.
```
Comment (valkyrie69) [inline modules/ROOT/pages/_partials/acb-reusable-steps.adoc]:
```suggestion
* If the test succeeds, you see a *Connection is valid* message.
```

Comment (valkyrie69) [reply modules/ROOT/pages/_partials/acb-reusable-steps.adoc]: Style guide says to omit "popup" and just refer to the element (message)

File: modules/ROOT/pages/_partials/acb-reusable-steps.adoc [hunk 2]
```diff
@@ -132,16 +147,15 @@ Snippets: Configure User Snippets
...
 // end::exchange-urls[]
-
-
-
-
+//
+//
 // tag::no-project-name-reuse[]
 // Cannot reuse project name
 To reuse an existing project name, you must delete the project first. See
```
Comment (valkyrie69) [inline modules/ROOT/pages/_partials/acb-reusable-steps.adoc]:
```suggestion
To reuse an existing project name, you must delete the project that is already using the name first. See
```

File: modules/ROOT/pages/_partials/acb-reusable-steps.adoc [hunk 3]
```diff
@@ -173,19 +188,50 @@ image:icon-api-console.png["",14,14] (*API Console*) icon to open the fragment i
...
+. Click the image:icon-start-debug.png["",15,15] (*Start Debugging (F5)*) icon for *Debug Mule Application*.
++
+Alternatively, if the *Start Debugging (F5)* icon is not present:
 
+.. Click *Run and Debug*. 
+.. If a *Select debugger* dropdown menu appears, select *Mule Xml Debugger* to initiate the debugging session.
++
+image::anypoint-code-builder::run-debug-imported-folder.png["Run and Debug button and Mule XML Debugger menu item"]
++
+Make sure that the application deploys successfully with the `DEPLOYED` log message in the output panel:
```
Comment (valkyrie69) [inline modules/ROOT/pages/_partials/acb-reusable-steps.adoc]:
```suggestion
When the application deploys successfully, you see a `DEPLOYED` log message in the output panel:
```

File: modules/ROOT/pages/_partials/acb-reusable-steps.adoc [hunk 4]
```diff
@@ -173,19 +188,50 @@ image:icon-api-console.png["",14,14] (*API Console*) icon to open the fragment i
...
+----
+// end::deploy-locally[]
+//
+
+//
+// tag::open-terminal[]
+. From your IDE, open the Terminal window in the console.
+
+* From the desktop IDE: Navigate to *View* > *Terminal*.
+* From the cloud IDE: Navigate to the menu icon > *Terminal* > *New Terminal*.
```
Comment (valkyrie69) [inline modules/ROOT/pages/_partials/acb-reusable-steps.adoc:234]: Do you have a menu icon, or we just assume everyone knows what a menu icon looks like?

File: modules/ROOT/pages/int-trigger-flows.adoc [hunk 5]
```diff
@@ -4,84 +4,139 @@
 
 include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"]
 
-//LOGO (web, desktop, or both)
-// include::partial$acb-ide-logos.adoc[tags="both-ides"]
+When an integration or implementation that you are developing builds successfully within the Anypoint Code Builder IDE, the application deploys locally, within your development environment, whether on your desktop or the cloud-based host for your applications.
```
Comment (valkyrie69) [inline modules/ROOT/pages/int-trigger-flows.adoc]:
```suggestion
When your integration or implementation builds successfully within the Anypoint Code Builder IDE, the application deploys to your local development environment, whether you are using the desktop or cloud-based host for your applications.
```

File: modules/ROOT/pages/int-trigger-flows.adoc [hunk 6]
```diff
@@ -4,84 +4,139 @@
...
-
-== Before You Begin
+[[trigger-urls]]
+== URLs for Triggering a Flow from an HTTP Listener
 
-* xref:start-acb.adoc[Set up and access the web or desktop IDE].
-* xref:int-create-integrations.adoc[Create an integration].
+For integrations and implementations that you build and deploy within your IDE during the development process, the URL to use for triggering an HTTP Listener differs depending on whether the listener is within the configuration XML for a scaffolded interface that is generated from an API specification or within a configuration XML file that does not implement a scaffolded interface.
 
+Typically, apps that use an HTTP Listener configure the host to `0.0.0.0` and the port to `8081` or a similar port, such as `8082`, if `8081` is currently in use by another application.
```
Comment (valkyrie69) [inline modules/ROOT/pages/int-trigger-flows.adoc]:
```suggestion
Typically, for apps that use an HTTP Listener you configure the host to `0.0.0.0` and the port to `8081`. If port `8081` is in use by another application, you can use a similar port, such as `8082`.
```

File: modules/ROOT/pages/int-trigger-flows.adoc [hunk 7]
```diff
@@ -4,84 +4,139 @@
...
+    <http:listener path="mypath" config-ref="inbound-request" doc:name="HTTP /flights" />
+    ...
+</flow>
+----
 
-. Open the Mule project that you want to test and navigate to *Run* > *Start Debugging* (`F5`).
-. After your application deploys successfully, open a new tab in your browser.
-. Type the URL of your cloud IDE instance, add the `/proxy/` path, and then add the port number you configured as a new path:
-`+https://<web-IDE-instance>/api/<port-you-configured-on-your-listener>/+`
+[[trigger-url-interface]]
```
Comment (valkyrie69) [inline modules/ROOT/pages/int-trigger-flows.adoc:47]: Is there a missing header?

File: modules/ROOT/pages/int-trigger-flows.adoc [hunk 8]
```diff
@@ -4,84 +4,139 @@
...
+* Trigger a flow from listener within a scaffolded interface by using a URL similar to this one from your IDE's terminal:
```
Comment (valkyrie69) [inline modules/ROOT/pages/int-trigger-flows.adoc]:
```suggestion
* Trigger a flow from the listener within a scaffolded interface by using a URL similar to this one from your IDE's terminal: 
```

File: modules/ROOT/pages/int-trigger-flows.adoc [hunk 9]
```diff
@@ -4,84 +4,139 @@
...
-. Search for the term _proxysession_:
-+
-image::application-storage-cookies.png[]
-. Copy the cookie name and value. For example:
+[[trigger-flow]]
+== Trigger a Flow
+
+Use the terminal in Anypoint Code Builder to trigger a flow through an HTTP Listener in an integration or implementation project that is deployed locally, within your IDE. This procedure is for Mule apps within the desktop or cloud IDE. 
+
+TIP: The examples in the procedure show how to trigger the flow with a `curl` command from the IDE's terminal. You can also use Thunder Cloud, a third-party extension to VS Code, to trigger a flow with the same URLs. For security, external REST clients, such as Advanced REST Client and Postman, are not allowed to send requests to projects in the cloud IDE.
```
Comment (valkyrie69) [inline modules/ROOT/pages/int-trigger-flows.adoc]:
```suggestion
TIP: The examples in this procedure show how to trigger the flow with a `curl` command from the IDE's terminal. You can also use Thunder Cloud, a third-party extension for VS Code, to trigger a flow with the same URLs. For security reasons, external REST clients, such as Advanced REST Client and Postman, are not allowed to send requests to projects in the cloud IDE.
```

File: modules/ROOT/pages/int-trigger-flows.adoc [hunk 10]
```diff
@@ -4,84 +4,139 @@
...
+
+Before attempting to trigger a flow, review <<trigger-urls>>, and be sure to meet the prerequisites in <<before-you-begin>>.
```
Comment (valkyrie69) [inline modules/ROOT/pages/int-trigger-flows.adoc]:
```suggestion
Before attempting to trigger a flow, review <<trigger-urls>>, and verify that you meet the prerequisites in <<before-you-begin>>.
```

File: modules/ROOT/pages/int-trigger-flows.adoc [hunk 11]
```diff
@@ -4,84 +4,139 @@
...
+include::anypoint-code-builder::partial$acb-reusable-steps.adoc[tag="open-terminal"]
+. In the IDE's terminal, click the name of the shell to use, such as PowerShell (for Windows), Bash (`bash`), or Z shell (`zsh` for Mac).
 +
-[source]
---
-proxysession5f5648c7115f4e2e8f3f9396355ac4e8-0=xxxxxxxxxxxxxx
---
+Alternatively, click the *+* icon in the terminal to open a new command prompt.
+. At the command prompt, provide a `curl` command to trigger the flow, for example:
+
```
Comment (valkyrie69) [inline modules/ROOT/pages/int-trigger-flows.adoc:126]: Is the example missing? Or is this a duplicated line?

File: modules/ROOT/pages/int-trigger-flows.adoc [hunk 12]
```diff
@@ -4,84 +4,139 @@
...
+* To trigger a flow from an HTTP listener that does not reside within an interface, use `curl` with a URL similar to this one:
```
Comment (valkyrie69) [inline modules/ROOT/pages/int-trigger-flows.adoc]:
```suggestion
* To trigger a flow from an HTTP Listener that doesn't reside within an interface, use `curl` with a URL similar to this one:
```

File: modules/ROOT/pages/start-add-folders.adoc [hunk 13]
```diff
@@ -61,13 +61,9 @@ image::file-open-folder-command.png[]
...
-. Open the Terminal window in the console.
-
-* From the desktop IDE: Navigate to *View* > *Terminal*.
-* From the cloud IDE: Navigate to the menu icon > *Terminal* > *New Terminal*.
-* For either IDE: Press Ctrl and then press the backtick key (`).
-
-. Provide the following command:
+//open terminal:
+include::anypoint-code-builder::partial$acb-reusable-steps.adoc[tag="open-terminal"]
+. Provide this command to create a folder:
```
Comment (valkyrie69) [inline modules/ROOT/pages/start-add-folders.adoc:66]: Need to add the command?

Comment (dukesphere) [reply modules/ROOT/pages/start-add-folders.adoc:66]: It's below in the  source so they can copy and edit it.

File: modules/ROOT/pages/_partials/acb-reusable-steps.adoc (hunk 4 above)
Comment (dukesphere) [reply modules/ROOT/pages/_partials/acb-reusable-steps.adoc:234]: I don't have an icon. It's that hamburger menu. I think we just call it the menu icon. Do we provide a graphic for it usually?

File: modules/ROOT/pages/int-trigger-flows.adoc (hunk 7 above)
Comment (dukesphere) [reply modules/ROOT/pages/int-trigger-flows.adoc:47]: Theres no header for this. I link to this bullet from elsewhere later.

File: modules/ROOT/pages/_partials/acb-reusable-steps.adoc (hunk 4 above)
Comment (dukesphere) [reply modules/ROOT/pages/_partials/acb-reusable-steps.adoc:234]: I have wondered if we should though... ¯\_(ツ)_/¯

File: modules/ROOT/pages/int-trigger-flows.adoc [hunk 14]
```diff
@@ -4,84 +4,139 @@
...
+//open terminal
+include::anypoint-code-builder::partial$acb-reusable-steps.adoc[tag="open-terminal"]
+. In the IDE's terminal, click the name of the shell to use, such as PowerShell (for Windows), Bash (`bash`), or Z shell (`zsh` for Mac).
 +
-[source]
---
-proxysession5f5648c7115f4e2e8f3f9396355ac4e8-0=xxxxxxxxxxxxxx
---
+Alternatively, click the *+* icon in the terminal to open a new command prompt.
+. At the command prompt, provide a `curl` command to trigger the flow, for example:
```
Comment (dukesphere) [inline modules/ROOT/pages/int-trigger-flows.adoc]:
```suggestion
. At the command prompt, provide a `curl` command to trigger the flow.
```

File: modules/ROOT/pages/int-trigger-flows.adoc (hunk 11 above)
Comment (dukesphere) [reply modules/ROOT/pages/int-trigger-flows.adoc:126]: fixed, thx for heads up

**Pull request merged**